### ISSUE TITLE
Shattered Pixel Dungeon now flagged as having "Anti-Features" on F-Droid

### ISSUE THREAD

**Issue by sviscapi** [00-Evan/shattered-pixel-dungeon#479]
Dear @00-Evan ,

Shattered Pixel Dungeon is now flagged as having "Anti-Features" on F-Droid:

https://f-droid.org/en/packages/com.shatteredpixel.shatteredpixeldungeon/

https://f-droid.org/en/docs/Anti-Features/#DisabledAlgorithm
https://f-droid.org/en/docs/Anti-Features/#KnownVuln

Could you please have a look at it ? Is it related in any way to issues #476 and #459 ?

Best regards,

Samuel

**Comment by 00-Evan** [comment]
I've already asked about this [Here](https://gitlab.com/fdroid/fdroiddata/-/issues/2140) and so far have not gotten a response.

I've honestly got no idea why these warnings are there and partly suspect it's an error on F-Droid's end. 'insecure signature' makes no sense as F-Droid signs the code themselves, and the latest release is still plenty recent from a security standpoint. 'known vulnerability' isn't completely nonsensical as the app does make network requests, but I don't see how that really leads to any security issues as its just grabbing xml data and populating the game's ui based on it. Maybe it's because the blog updater can use HTTP in some cases? This only happens when the android device is likely to not support modern encryption though (android 4.4-).

Honestly all of this is making me strongly consider dropping F-Droid as an official platform for Shattered, meaning I would no longer mention/link to it on the website or GitHub releases. It just isn't practical to link to it on the website and release posts and then have there be a risk of the F-Droid version being 2+ weeks outdated because their server refuses to work with recent build tools, despite those tools being added in the code. Even if there are no issues the F-Droid version often lags behind other releases by many days. And now there's an additional risk of F-Droid claiming my app has security issues with no explanation or ability to react reasonably quickly on my end.

**Comment by marcoXbresciani** [comment]
I'd love to still see Shattered on F-Droid. Most of the applications I have come from there.

**Comment by 00-Evan** [comment]
I don't have any intentions of removing the game from F-Droid. It's technically independent of me so I probably couldn't even if I wanted to.

**Comment by marcoXbresciani** [comment]
Yup. My point was that even if it's lagging behind, I'd still prefer to download/update it from FDroid.
This said, if obviously it's a pain and a pointless burden for you to maintain it through those requirements, I'll be anyway glad to take it from Google... Even if I'll lose all my current statistics. 😉

**Comment by 00-Evan** [comment]
Ideally the F-Droid version will still be updated, assuming they manage to sort out whatever is stopping it. I'm just not going to official endorse or link to it anymore.

**Comment by 00-Evan** [comment]
Unfortunately Shattered v0.8.2d has also not been released on F-droid. I'm going to close this issue as there's nothing really to be done on my part, this is a problem on F-Droid's end.

**Comment by MiroslavR** [comment]
0.8.2d is actually available in the F-Droid Archive repo, which can be enabled from Settings→Repositories. It looks like they're trying to figure out why new releases end up in the archive instead of the normal repo: https://gitlab.com/fdroid/fdroidserver/-/issues/824

**Comment by sviscapi** [comment]
Hi all,
@MiroslavR : thanks for the heads-up, the latest version is indeed available on F-Droid after enabling the Archive repository.
Cheers, Sam
